### ISSUE TITLE
Quest "The Balance of Light and Shadow"

### ISSUE THREAD

**Issue by Drepplin** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#112]
The quest is way more difficult than it used to be before and I know several priest waiting for this to get fixed. The waves spawn too fast compared to retail and the mobs get too much damage too quickly.

**Comment by Nzzoth** [comment]
Before the script for the quest didn't exist, you just killed the NPC and received your reward.

The quest is identical to what we have watched on multiple youtube videos and it has been completed several times in a far lesser gear than preraid BIS before it was released onto live servers.

I also know for a fact many people actually finished the quest and it's a "EPIC" quest therefor it's suppose to be hard.

Issue closed.

**Comment by Drepplin** [comment]
Identical to what? Blizzlike or like other private servers?
In retail, the skeletons attack the mobs, not only the player. There was also a short break between the mobs. There was also a possibility for a shaman or a paladin to assist via totems and auras. Now the cleaner showes up when someone is far far away.
https://www.youtube.com/watch?v=oN-WH95QRy8 
In this video from retail you see the brake between the waves, and the waves who are spawning on warsong, are nothing like this. Currently next waves spawn the others are not past Eris yet.
Now I have around 10.000 mana when fully buffed and I cannot get past wave 4, in the youtube videos most priest have half that.
https://www.youtube.com/watch?v=loroyK2RfnQ  15:50
This video is from annother private server. There the skeletons sometime spawn before the waves(when there is suppose to be a break) which makes it alot easier to handle healing the mobs.
You know for a fact there are many people who have finished this quest. Well I don't know anyone who has done it since I got my eye of divinity. But I am sure there are people who has completed this quest. As you must know there are some players on this server who are really good, some of them so good they don't have a social life. Is this server only for people who are professionals?
I have talked to several priest who have done this before, and they all agree that this is much harder than it used to be. It is really sad that all those eye of divinities are lying around in our bags because you won't acknowledge the quest is not working properly.

**Comment by Drepplin** [comment]
also, Spirit tap does not work when killing the mobs, as it should

**Comment by higuysitme** [comment]
I tried on my priest with bis preraid 9k mana and you cant really do it. I just wasted 300g flasks/mana pots etc. It seems like you tested it out on ptr with full t3/naxx gear. It should not be that hard to complete as it is right now. What supposed to do average player? Have you seen videos of clickers in gear with no bonus healing doing it?
1)Humanoids that you supposed to heal moving too fast.
2)Humanoids losing hp too fast that you cant even leave them to start healing second wave.
3)Swarms of skeletons instead of 3-4 makes it really hard.
